### ISSUE TITLE
Do proxyman have transform response feature?

### ISSUE THREAD

**Issue by thepinyo** [ProxymanApp/Proxyman#219]

**Comment by NghiaTranUIT** [comment]
Yes, it's https://github.com/ProxymanApp/Proxyman/issues/118, which is our road map. It will release on Proxyman 1.4.6. Stay tuned 🎉

**Comment by thepinyo** [comment]
@NghiaTranUIT good, i will await you.

**Comment by NghiaTranUIT** [comment]
@thepinyo  Just friendly reminder that simple version of Remap is "Remap response from file" #224  will be implemented in next version Proxyman 1.4.6 👍

**Comment by NghiaTranUIT** [comment]
Hey @thepinyo,

It just post the BETA of this feature here https://github.com/ProxymanApp/Proxyman/issues/224#issuecomment-517331332. Let try it and let me know if you have any issues 🌮

**Comment by thepinyo** [comment]
@NghiaTranUIT oh thank i'll try.

**Comment by thepinyo** [comment]
@NghiaTranUIT oh thank i'll try.

**Comment by NghiaTranUIT** [comment]
@thepinyo the [Proxyman 1.4.7](https://github.com/ProxymanApp/Proxyman/releases/tag/1.4.7) has officially been released now. Let check it out 👍

**Comment by thepinyo** [comment]
how to change http status code of response.

**Comment by NghiaTranUIT** [comment]
Hey @thepinyo 

Currently, the Local map is not capable of manipulating the Status code of response. It's must be a ReWrite feature.

We're working the Breakpoint (able to change the Request data), then moving on to the ReWrite in next versions.

**Comment by thepinyo** [comment]
good job bro, fighting 😉

**Comment by NghiaTranUIT** [comment]
Hey @thepinyo, let check out Breakpoint on Proxyman 1.5.0 https://github.com/ProxymanApp/Proxyman/releases/tag/1.5.0

Get ready for the Breakpoint 😍

**Comment by robertoseidenberg** [comment]
Is it possible that this feature is now broken? 🤔 
Im using Proxyman 1.23.0 (12300) and I have not been able to get a breakpoint to fire on the response. If I put a breakpoint on an url for "request" and "response" Proxyman breaks two times but the UI always only shows the "request" data. Hence no way to edit the response.

**Comment by NghiaTranUIT** [comment]
Hey @robertoseidenberg , can you share your screenshot when the Response panel doesn't appear on Breakpoint?

Here is how I set a breakpoint on a Response

<img width="1997" alt="Screen_Shot_2020-05-06_at_23_35_22" src="https://user-images.githubusercontent.com/5878421/81203651-648c5f00-8ff2-11ea-9615-ac23da512dde.png">

**Comment by robertoseidenberg** [comment]
Made you a .gif. I do remember having seen this on my computer as well but now it does not show up any more. Can not really say when it changed though. I'm sorry.

![ProxymanBreakpoint](https://user-images.githubusercontent.com/699762/81208589-9b04b280-8fcf-11ea-87ff-fe23316b1d15.gif)

**Comment by NghiaTranUIT** [comment]
Look like it’s a UI bug. Pls help me to reset the app from Help menu > Advanced > Reset
Then open on the big window size

**Comment by NghiaTranUIT** [comment]
I will investigate tomorrow 👍

**Comment by NghiaTranUIT** [comment]
Confirmed that it's a UI glitch if the size of Breakpoint windows is small. You can fix it by opening a big size and reset the app (Help -> Advanced -> Reset) @robertoseidenberg

**Comment by robertoseidenberg** [comment]
Hey @NghiaTranUIT, sorry for the delay. I can confirm it's an UI bug. I did reset the app and fullsized the window and that helped. Thanks a lot for you fast response! App is awesome by the way. 😏

**Comment by NghiaTranUIT** [comment]
Glad to hear that. I will fix it soon 👍

**Comment by robertoseidenberg** [comment]
Thanks! The workaround works well enough for me meanwhile. I'm just super happy the feature is back. 😄
